src/components/LargeText.js

```diff
@@ -5,7 +5,10 @@ import colors from '../assets/colors.js'
 class LargeText extends React.Component {
   render() {
     return (
-      <Text style={styles.text}>{this.props.text}</Text>
+      // Seth - Using the array notation here for style allows us to pass
+      //        styles in from the props and combine them with the inline
+      //        styles defined in this components stylesheet.
```

Comment:
typically you wouldn't leave a comment on framework fundamentals, but in this case it could be useful as we are all learning react native

Reply:
Yes, this is why I have it in here.